.gitignore

```diff
@@ -4,8 +4,7 @@ build/
 dist/
 docs/
 *.autosave
-bin/macosx64/Sparkle.framework
-bin/macosx64/borg
+bin/*
```

Comment:
assuming that the dist/ and docs/ work, no need for the `*`?

Reply:
is docs/ something you are still working on or why is it here?

Reply:
That's true. And they seem to work. 🎉